### ISSUE TITLE
Passing more info with requests to resolver

### ISSUE THREAD

**Issue by RafalWilinski** [michalkvasnicak/aws-lambda-graphql#4]
Thanks for this awesome module! Works great for me, however, I wasn't able to implement one thing.

Let's assume that header `X-User-ID` comes with a request. How can I pass it down to resolver?

**Comment by michalkvasnicak** [comment]
That's good question. Perhaps we could pass whole AWS API Gateway event to context handler?

It could work similarly as in `graphql-yoga` that you will have access to request in [context](https://github.com/prisma/graphql-yoga#api)

**Comment by RafalWilinski** [comment]
Can you please give me some guidance here?

I've tried passing custom context from `httpHandler` function level, however, it does not work. Only passing custom `context` arg inside `createHttpHandler` calls sets it. My goal however is to call `createHttpHandler` just once and re-use it with concurrent invocations.

```
const httpHandler = createHttpHandler({
  connectionManager,
  schema,
});

async function _handler(
  event: APIGatewayEvent | APIGatewayWebSocketEvent | DynamoDBStreamEvent,
  ctx: APIGatewayEventRequestContext | any,
) {

  return httpHandler(
    event as any,
    {
      abc: '123',
      context: {
         ...ctx,
        abc: '1234',
      },
    } as any,
    null as any,
  );
});

```

☝️ None of that works.

**Comment by michalkvasnicak** [comment]
@RafalWilinski yeah this is by design because I haven't expected the usage as you have :) But is fairly easy to implement that feature here https://github.com/michalkvasnicak/aws-lambda-graphql/blob/master/packages/aws-lambda-graphql/src/createHttpHandler.ts#L64

But another question is, isn't a context creator function enough for that use case? See https://github.com/michalkvasnicak/aws-lambda-graphql/blob/0ec83695cb9355ead41b816a81b91f6fca0663fc/packages/aws-lambda-graphql/src/execute.ts#L30

**Comment by RafalWilinski** [comment]
> isn't a context creator function enough for that use case?

Not in my case. The `httpHandler` persists between invocations and I need `awsRequestId` which is unique per handler call. 

Should I create a PR to include this functionality, or in another words, would you accept such functionality or should I create a fork?

**Comment by michalkvasnicak** [comment]
Oh I see, the `awsRequestId` is available in lambda's context argument. In that case could it be enough for your use case if the library managed to pass you a lambda context to context creator too? The same thing as an event is passed to context creator.

**Comment by RafalWilinski** [comment]
> could it be enough for your use case if the library managed to pass you a lambda context to context creator too? 

Yes, exactly!

**Comment by michalkvasnicak** [comment]
@RafalWilinski implemented in #23, releaed as `0.7.0`
